src/utils/daos_hdlr.c

```diff
@@ -3812,14 +3958,15 @@ parse_acl_file(const char *path, struct daos_acl **acl)
 
 	instream = fopen(path, "r");
 	if (instream == NULL) {
-		fprintf(stderr, "Unable to read ACL input file '%s': %s\n",
+		fprintf(ap->errstream,
+			"Unable to read ACL input file '%s': %s\n",
 			path, strerror(errno));
 		return daos_errno2der(errno);
```

Comment:
Not a regression but unsafe errno handling here, fprintf() may well have modified it.